app/src/routes/api/v1/gfw.router.ts

```diff
@@ -0,0 +1,248 @@
+import { Context, Next } from 'koa';
+import Router from 'koa-router';
+import {
+    RWAPIMicroservice,
+    RequestToMicroserviceOptions,
+} from 'rw-api-microservice-node';
+import type request from 'request';
+
+import FieldSerializer from 'serializers/field.serializer';
+import logger from 'logger';
+import DatasetMiddleware from 'middleware/dataset.middleware';
+import GfwService from 'services/gfw.service';
+import ErrorSerializer from 'serializers/errorSerializer';
+
+const API_VERSION: string = 'v1';
+
+const router: Router = new Router({
+    prefix: `/api/${API_VERSION}/gfw`,
+});
+
+class GfwRouter {
+    static getCloneUrl(url: string, idDataset: string): Record<string, any> {
+        return {
+            http_method: 'POST',
+            url: `/dataset/${idDataset}/clone`,
+            body: {
+                dataset: {
+                    datasetUrl: url
+                        .replace('/gfw', '')
```

Comment:
replace with `.replace('/gfw/', '/')` and make the equivalent change on the line below

Reply:
Below is an example result of the existing code which looks right. Am I missing something as to why the change you suggest is needed?

```javascript
const datasetUrl = '/api/v1/gfw/query/1639574869912?sql=SELECT%20%2A%20from%20DATA%20LIMIT%202'
const resultUrl = datasetUrl.replace('/gfw', '').replace('/api/v1', '')
resultUrl
> '/query/1639574869912?sql=SELECT%20%2A%20from%20DATA%20LIMIT%202"
```